src/index.js

```diff
@@ -1,7 +1,48 @@
 /*eslint no-unused-vars: "off"*/
-import React from 'react';
+import React, { Component } from 'react';
+import { render } from 'react-dom';
 import ReactDom from 'react-dom';
+import './index.scss';
+import ListItems from './ListItems.js';
+import TodoInputField from './TodoInputField.js';
 
-require('!style-loader!css-loader!sass-loader!./index.scss');
+class App extends Component {
+  constructor() {
+    super();
 
-ReactDom.render(<div>Hello World!</div>, document.getElementById('root'));
+    this.state = {
+      list: []
+    };
+    this.onSave = this.onSave.bind(this);
+    this.onClear = this.onClear.bind(this);
+    this.onCompleted = this.onCompleted.bind(this);
+  }
+
+  onSave(text) {
+    this.setState({
+      list: [...this.state.list, {todo: text, completed: false}]
+    });
+  }
+  onClear() {
+    this.setState({
+      list: []
+    });
+  }
+  onCompleted(idx) {
+    let newList = this.state.list;
+    newList[idx].completed = !newList[idx].completed;
```

Comment:
Nicely done! This is a great implementation of toggling a boolean state on an object. You'll see these toggle functions everywhere in React development